src/uv_encoding.c

```diff
@@ -408,6 +415,7 @@ static int decodeAppendEntries(const uv_buf_t *buf,
 
     cursor = buf->base;
 
+    args->version = 1;
```

Comment:
Shouldn't we set the initial version to `0`? In this way the wire format what we intend to use in the future is already correct (I'm referring to the idea of encoding the version in the message preamble).

I'm also fine setting the initial version to `1`, I guess we'll just have to remember to convert `0` to `1` whenever we'll switch to the new wire format. I'm just pointing this out to make sure we're not missing something.